### DISCUSSION TITLE
Wunsch: Dynamic Power Limiter Zeitvorgabe

### DISCUSSION THREAD

**Discussion by iroc-fabi** [hoylabs/OpenDTU-OnBattery#1285] [General]
Es wäre schön, wenn man die Zeit vorgeben könnte, an denen Strom aus dem Akku gezogen werden soll. z.B von 18 bis 6 Uhr. 
So kann der WR immer eingeschaltet bleiben und ich muss ihn nicht per Shelly an- und abschalten.

**Comment by Manos1966** [comment]
Gibt es ein Nachteil ihn per Shelly an-/abzuschallten?

Es ist die einfachste und beste Loesung.

PS. und vor allem, eine extra Sicherung, ob der Wechselrichter funktioniert und wieviel er produziert (bei Shelly PM)

**Comment by Sorbat** [comment]
Es geht ihm wohl um das Vorladen der Kondensatoren.

Aber warum wird der Wechselrichter nicht softwareseitig mit Hilfe eines Powermeters,welches den tatsächlichen Bedarf ermittelt, geregelt?

**Reply by Manos1966** [reply]
Ah soooo!

Du benutzt aber die SHELLY auf der AC Seite, oder?
Warum werden die Kondensatoren auf der DC Seite leer, wenn du die AC-Seite abschaltest?
(Antwort: werden sie nicht 😉  )

**Reply by spcqike** [reply]
Vorladen ist egal wenn du ihn AC seitig trennst. Da bleibt er am DC Eingang ja dauerhaft am Strom. 

Nachteil der AC Trennung ist die initiale Netzsynchronisation. Das dauert ja dann erst mal ein paar Minuten. Bei meinem HMS-2000 bspw 10-15 Minuten, bis er mal 2kW liefert. (Sync und ramp up)

**Comment by Manos1966** [comment]
Um Missverstaendnisse zu vermeiden: Die Idee ist nicht schlecht.

ABER,

Die Möglichkeit, fuer 13EUR, eine SHELLY PM am AC Ausgang der HOYMILES zu haben als zusätzliche "Kontrol-Instanz" mit Zugriffsmoeglichkeit von **Überall (Shelly Cloud)** um:
- Leistung zu monitoren und 
- bei Bedarf den Wechselrichter zu trennen (falls die DTU nicht das macht, was sie machen sollte) <br>
**UNBEZAHLBAR**! 😆 

Ich habe sogar Installationen wo ich eine Shelly 2.5 nutze: Eine Seite fuer den Wechselrichter, die andere um die USB Versorgung der DTU ein/ausschalten falls die DTU nicht mehr erreichbar ist (heutzutage ist die DTU stabil, man braucht es nicht mehr, andererseits, man weiss ja nie... 😉 ).

**Comment by schlimmchen** [comment]
> Es wäre schön, wenn man die Zeit vorgeben könnte, an denen Strom aus dem Akku gezogen werden soll. z.B von 18 bis 6 Uhr.

Warum?

> So kann der WR immer eingeschaltet bleiben und ich muss ihn nicht per Shelly an- und abschalten.

Warum tust du das?

**Reply by Ralf-64** [reply]
Hallo Fabi,
weiß nun nicht was du an Ausrüstung hast...
Mit Node-Red kann man solche "Gimmicks" recht einfach erstellen. Es gibt dort z.b. eine Node "Sunrise-Sunset"... Auch wäre damit der Weg zu Grafanna geöffnet, falls du visualisieren möchtest...
LG Ralf

**Comment by Ralf-64** [comment]
![20240915_231030](https://github.com/user-attachments/assets/5b31bd78-856a-4ec7-9054-1f39c93cb4b0)

So könnte das dann aussehen. 
An dieser Anlage läuft ein Inverter den ganzen Tag auf Volleinspeisung. Abends/ Nachts übernimmt dann ein 2ter (Akku)Inverter in DPL Regelung den benötigten Verbrauch -> blauer Graph...
LG Ralf
